scripts/composer/AssetInstaller.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * @file
+ * Contains \DrupalProject\composer\AssetInstaller.
+ */
+
+namespace DrupalProject\composer;
+
+use Composer\Script\Event;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class AssetInstaller {
+
+  public static $assetFileTypes = [
+    '.htaccess',
+    '*.css',
+    '*.ico',
+    '*.jpeg',
+    '*.jpg',
+    '*.js',
+    '*.png',
+    '*.svg'
+  ];
+
+  public static $frontControllers = [
+    'index.php',
+    'core/install.php',
+    'core/rebuild.php',
+    'core/modules/statistics/statistics.php',
+  ];
+
+  public static function install(Event $event) {
+    $extra = $event->getComposer()->getPackage()->getExtra();
+    if (!isset($extra['drupal-app-dir'])) {
+      throw new \RuntimeException('Please configure drupal-app-dir in your composer.json');
+    }
+    if (!isset($extra['drupal-web-dir'])) {
+      throw new \RuntimeException('Please configure drupal-web-dir in your composer.json');
+    }
+    foreach (static::$frontControllers as $fileName) {
+      static::createStubPhpFile($extra['drupal-app-dir'], $extra['drupal-web-dir'], $fileName);
```

Comment:
Before call `createStubPhpFile()`, the web and app folders must exist, otherwise the process will return an error
```[Symfony\Component\Filesystem\Exception\IOException]  
  Unable to write to the "/" directory.   ```

This was not happening before because `createSymlinks()` was being called before `createStubPhpFile()` and creating the folders.
https://github.com/drupal-composer/drupal-project/pull/180/commits/b111239718037aad21693492b035f9a3ee3758cf#diff-a97cd408fd45330f7510ba6d71ab4907R25

Suggestion:
```
// Ensure that the app and web directories exist.
$cfs = new \Composer\Util\Filesystem();
$cfs->ensureDirectoryExists($web_dir);
$cfs->ensureDirectoryExists($app_dir);

// Call createStubPhpFile().

//  Call createSymlinks().
```